File: LV0/특이한 정렬.js
function solution(numlist, n) {
  numlist.sort((a, b) => {
    const newA = Math.abs(a - n);
    const newB = Math.abs(b - n);
    if (newA === newB) {
      return b - a;
    }
    return newA - newB;
  });
  return numlist;
}

File: LV0/특이한 정렬.java
class Solution {

  public int[] solution(int[] numlist, int n) {
    Arrays.sort(
      numlist,
      (a, b) -> {
        int newA = Math.abs(a - n);
        int newB = Math.abs(b - n);
        if (newA == newB) {
          return a - b;
        }
        return newA - newB;
      }
    );
    return numlist;
  }
}
